cns/restserver/restserver.go

```diff
@@ -1573,3 +1576,32 @@ func (service *HTTPRestService) getNetPluginDetails() *networkcontainers.NetPlug
 	configPath, _ := service.GetOption(acn.OptNetPluginConfigFile).(string)
 	return networkcontainers.NewNetPluginConfiguration(pluginBinPath, configPath)
 }
+
+// Retrieves the number of logic processors on a node. It will be primarily
+// used to enforce per VM delegated NIC limit by DNC.
+func (service *HTTPRestService) getNumberOfCPUCores(w http.ResponseWriter, r *http.Request) {
+	log.Printf("[Azure-CNS] getNumberOfCPUCores")
+	log.Request(service.Name, "getNumberOfCPUCores", nil)
+
+	var num int
```

Comment:
Can you combine the vars together ?
var (
num int
returnCode int
errMsg string
)

Reply:
:-). Is this being done anywhere in similar context?

Reply:
I agree its not done elsewhere but whatever ashvin suggested looks better. lets do this from now on